docs/tutorials/fundamentals/tutorial-token-transfer-and-nft-purchase.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Tutorial - Token transfer and Uniq purchase transactions'
+order: -99998
+outline: [0, 5]
+---
+
+# Tutorial - Tutorial - Token transfer and Uniq purchase transactions
+
+This tutorial will demonstrate how to use the Ultra toolkit for sending asset (transfer) and purchasing Uniq interact with Ultra the blockchain.
+
+## Prerequisites
+
+Before starting this tutorial, you will need:
+
+- A developer account on the Ultra testnet. If you don't have one, follow our previous tutorial: [Generate a key and create a developer Testnet account](./tutorial-generate-key-and-create-testnet-account.md).
+- Your account must have sufficient UOS tokens for transactions and fees.
+- The latest version of the Ultra toolkit installed on your machine.
```

Comment:
I think this prerequisite is not required since we expect the user to use toolkit hosted at https://toolkit.ultra.io/